pkg/splunk/enterprise/clustermaster.go

```diff
@@ -195,9 +200,11 @@ func ApplyClusterMaster(ctx context.Context, client splcommon.ControllerClient,
 			}
 		}
 
+		// Create podExecClient
+		podExecClient := splutil.GetPodExecClient(client, cr, "")
+
```

Comment:
any specific reason for moving it out?

Reply:
This is direct port of fix from Arjun here : https://github.com/splunk/splunk-operator/pull/1053

Reply:
@gaurav-splunk @pdhanoya-splunk This change is not required. It was a part of an earlier logic workflow which was changed later due to a review comment. IMO to maintain symmetry we can keep it as is for now.